src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -99,6 +99,24 @@ GARY_IMPL(VMHELPDEF, hlpDynamicFuncTable, DYNAMIC_CORINFO_HELP_COUNT);
 
 #else // DACCESS_COMPILE
 
+uint64_t g_cbILJitted = 0;
+uint32_t g_cMethodsJitted = 0;
+
+FCIMPL0(INT64, GetJittedBytes)
+{
+    FCALL_CONTRACT;
+    return g_cbILJitted;
+}
+FCIMPLEND
+
+FCIMPL0(INT32, GetJittedMethodsCount)
+{
+    FCALL_CONTRACT;
+
```

Comment:
Nit: Make the formatting the same for the both methods (a new line before return).